src/screens/Requests/Requests.tsx

```diff
@@ -270,15 +311,31 @@ const Requests = (): JSX.Element => {
           </div>
           <div className={styles.btnsBlock}>
             <div className="d-flex">
-              <Dropdown aria-expanded="false" title="Sort organizations">
-                <Dropdown.Toggle variant="outline-success">
+              <Dropdown
+                aria-expanded="false"
+                title="Sort organizations"
+                data-testid="sort"
+              >
+                <Dropdown.Toggle
+                  variant="outline-success"
+                  data-testid="sortuser"
+                >
                   <SortIcon className={'me-1'} />
                   {t('sort')}
```

Comment:
Here instead of `Sort` text you should show the selected option, and the sort button should become of `variant='success'` meaning sorting is active now

Reply:
![image](https://github.com/PalisadoesFoundation/talawa-admin/assets/97614113/12c5574e-4802-46b7-a90e-c56ff68a9665)